### ISSUE TITLE
multi channel images

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-utils-python#40]
@martinschorb does the python library support multi channel images as an input? 
if so, what names does it give the individual channels?
(cc @K-Meech)

**Comment by constantinpape** [comment]
No, it currently doesn't support multi channel images. This would need to be handled by a user in preprocessing.

**Comment by martinschorb** [comment]
here's my hacks to enable the functionality to extract single channels from multi-channel stacks while importing them in MoBIE:

https://github.com/constantinpape/cluster_tools/pull/42

https://github.com/constantinpape/elf/pull/81

**Comment by martinschorb** [comment]
This works now (at least with OME-Zarr), correct?
Would it be relevant to adjust LUT in the corresponding views as well when adding such a source, or do we want to leave that up to the user?

**Comment by constantinpape** [comment]
I will follow up in https://github.com/mobie/mobie-viewer-fiji/issues/1093 to see if this works it all. Once we have figured that out we can see about the color maps (best to do that in a separate issue then.)

**Comment by martinschorb** [comment]
WIP to enable adding a single channel to MoBIE
https://github.com/mobie/mobie-utils-python/pull/130

We might be able to extend that to general multi-channel application.
